doc/source/getting_started/installing.rst

```diff
@@ -187,7 +187,7 @@ Installing the Debian packages
 
 ::
 
-   $ curl https://www.apache.org/dist/cassandra/KEYS | sudo apt-key add -
+   $ curl -L https://www.apache.org/dist/cassandra/KEYS | sudo apt-key add -
```

Comment:
rather than add the redirect, let's use the final destination.

eg
`curl https://downloads.apache.org/cassandra/KEYS | sudo apt-key add -`

And… can you base this against the cassandra-3.11 branch please. (It applies to all branches, but we only have docs currently for cassandra-3.11 and trunk)

Reply:
gladly

Reply:
should we keep the -L in case we redirect again in the future or do you think that's a security risk?

Reply:
Looks like this already got done in the 3.11 branch https://github.com/apache/cassandra/commit/c93c2983cd3f795ef81c7cd298a115e05a42ed5c#diff-6d9ad0776a3e0cdd1be608c3424af5b0

Reply:
and propagated to the website https://cassandra.apache.org/download/ so I'm closing this issue.